GOVERNANCE.md

```diff
@@ -0,0 +1,147 @@
+and valuable contributions to the Rez project to become a new
+Committer. New committers are approved by vote of the TSC.
+
+If you are interested in becoming a Committer, contact the TSC at:
+  rez-discussion@lists.aswf.io
+
+## Technical Steering Committee
+
+The Technical Steering Committee (TSC) oversees the overall technical
+direction of Rez, as defined in the project
+[charter](https://github.com/AcademySoftwareFoundation/foundation/blob/main/project_charters/rez-charter.pdf).
+This charter defines the TSC member terms and succession policies.
+
+The responsibilities of the TSC include:
+
+* Coordinating technical direction of the project.
+
+* Project governance and contribution policy.
+
+* GitHub repository administration.
+
+* Maintaining the list of additional Committers.
+
+* Appointing representatives to work with other open source or open
+  standards communities.
+
+* Discussions, seeking consensus, and where necessary, voting on
+  technical matters relating to the code base that affect multiple
+  projects.
+
+* Coordinating any marketing, events, or communications regarding the
+  project.
+
+The TSC elects a Chair person, who acts as the project manager,
+organizing meetings and providing oversight to project
+administration. The Chair is elected by the TSC. The Chair may
+also serve as the Rez representative on the Academy Software
+Foundation (ASWF) Technical Advisory Council (TAC), or one may
+be separately elected by the TSC. The Rez representative will
+represent the project at ASWF TAC meetings.
+
+### Current TSC Members
+
+Unless otherwise denoted with †, members of the TSC contribute to rez
+independent of their employer, but are listed regardless for transparency.
+
+* Allan Johns (chair, author) - NVIDIA
+* Brendan Abel - Walt Disney Imagineering
+* Jean-Christophe Morin - Signiant
+* Stephen Mackenzie - NVIDIA
+* Thorsten Kaufmann - Accenture Song Content
+
+### TSC Meetings
+
+All meetings of the TSC are open to participation by any member of the
+Rez community. Meeting times are listed in the [ASWF technical
+community calendar](https://lists.aswf.io/g/tac/calendar). The TSC
+Chair moderates the meeting, or appoints another TSC member to
+moderate in his or her absence.
+
+Items are added to the TSC agenda which are considered contentious or
+are modifications of governance, contribution policy, TSC membership,
+or release process, in addition to topics involving the high-level
+technical direction of the project.
+
+The intention of the agenda is not to approve or review all
+patches. That should happen continuously on GitHub and be handled by
+the larger group of Committers.
+
+Any community member or Contributor can ask that something be reviewed
+by the TSC at the meeting by logging a GitHub issue. Any Committer,
+TSC member can bring the issue to the TSC's
+attention by applying the `TSC` label.
+
+Prior to each TSC meeting, the meeting chair will share the agenda with members
```

Comment:
So far it's me that has announced the meetings (or even coordinated them). Should we instead say:

> Prior to each TSC meeting, a TSC member will share the agenda with members of the TSC. TSC members can also add items to the agenda at the beginning of each meeting. The meeting chair and the TSC cannot veto or remove items. The agenda will also be shared on Slack in the #rez channel, preferably a day before if possible.

Would that make sense?

Reply:
done.